flash/1.2.11/tests/test_controls.py

```diff
@@ -0,0 +1,23 @@
+import unittest
```

Comment:
You're just doing checksums with this script, right?

Have you looked at using md5sum and diff? Then you wouldn't need to install python.

I think I should try this for my next container so you can see an example.